jscomp/core/js_exp_make.ml

```diff
@@ -651,168 +658,168 @@ let rec not ({expression_desc; comment} as e : t) : t =
   | Bin(Gt,a,b) -> {e with expression_desc = Bin (Le,a,b)}
   | x -> {expression_desc = Caml_not e ; comment = None}
 and js_not ({expression_desc; comment} as e : t) origin : t =
-  match expression_desc with 
-  | Bin(EqEqEq , e0,e1) 
-    -> 
+  match expression_desc with
+  | Bin(EqEqEq , e0,e1)
+    ->
     bool_of_boolean {expression_desc = Bin(NotEqEq, e0,e1); comment}
-  | Bin(NotEqEq , e0,e1) -> 
+  | Bin(NotEqEq , e0,e1) ->
     bool_of_boolean {expression_desc = Bin(EqEqEq, e0,e1); comment}
-  | Bin(Lt, a, b) -> 
+  | Bin(Lt, a, b) ->
     bool_of_boolean {e with expression_desc = Bin (Ge,a,b)}
-  | Bin(Ge,a,b) -> 
+  | Bin(Ge,a,b) ->
     bool_of_boolean {e with expression_desc = Bin (Lt,a,b)}
-  | Bin(Le,a,b) -> 
+  | Bin(Le,a,b) ->
     bool_of_boolean {e with expression_desc = Bin (Gt,a,b)}
-  | Bin(Gt,a,b) -> 
+  | Bin(Gt,a,b) ->
     bool_of_boolean {e with expression_desc = Bin (Le,a,b)}
   | _ -> {expression_desc = Caml_not origin; comment = None}
 
 let rec ocaml_boolean_under_condition (b : t) =
```

Comment:
This can also be removed for cleanup: it's just the identity function.

Reply:
This is not ready for review. Actually I would like to keep this diff as small as possible, and do the clean up in a separate diff